app/views/_components/result-detail/template.njk

```diff
@@ -30,7 +32,7 @@
   {% endif %}
   {% if params.trainingLocation %}
     <div class="app-result-detail__row">
-      <dt class="app-result-detail__key">Training centre</dt>
+      <dt class="app-result-detail__key">University campus</dt>
```

Comment:
Is "campus" universally-applicable? I'm not sure I'd describe UCL as having a campus, for example, as I'd associate that with out-of-town Universities. "University location" maybe? 🤷‍♂️

Reply:
Fixed!